docs/requirements.txt

```diff
@@ -1,5 +1,6 @@
 # NumPy is required for building `bezier` from source.
 numpy >= 2.0.0
+setuptools >= 70.0.0
```

Comment:
This is for `doctest` and may be able to be done more close to doctests in `noxfile.py`